### DISCUSSION TITLE
Mapbox or Google Maps

### DISCUSSION THREAD

**Discussion by spykee2005** [BimmerGestalt/AAIdrive#553] [Q&A]
Hi All, 

This is quiet satisfying to see such motivation here.

I'm actually using custom navigation with Mapbox and I'd like to knowthe process to use Gmaps.

I can't find the way to do it.

Thank a lot and keep doing. That's awsome.

Spykee2005

**Comment by hufman** [comment]
Thank you so much for the kind words!
I'm glad you find the custom map useful! I can't provide builds with Gmaps enabled, but you can build it yourself with that option. Check out the [Developer Guide](https://bimmergestalt.github.io/AAIdrive/developers) for how to set up the build environment and the API key and that should get you started.

**Reply by spykee2005** [reply]
Thanks for your reply but i'm not so familier dith coding and staff like that. 😀

Does any one has already done this? 

Is it possible to share it?

Kind regards

SPYKEE2005

**Reply by hufman** [reply]
Good evening! Some people have done it, but they have not shared widely because the API is billable and doesn't have a very generous free tier, and this feature goes against the Google terms of service. With the new Mapbox build going public, almost all of the new development is focused on this version.
Why do you want to try the Google Maps build? It has the same featureset (actually, fewer features lately) as the Mapbox version, except using Google data sources. It can not be linked to your account, so it won't have your search history or your favorite places bookmarked. What are you looking to find in the gmap build?
